### ISSUE TITLE
Single bounding box multi label 

### ISSUE THREAD

**Issue by abhigoku10** [Megvii-BaseDetection/YOLOX#962]
@FateScript  hi i want to modify the yolox to perform the following requirements 
1. I want to perform person detection and obtain its class along with the gender/age and attributes custom annotation is available so can we modify the architecture accordingly as shown below 
![image](https://user-images.githubusercontent.com/22094053/144589725-be26fb12-a72f-4869-a3ee-69d13c2024b0.png)

2. can we add multitask branches to the existing architecture which performs the scene classification task 

Please share your thougths

**Comment by FateScript** [comment]
Yes, you could try such a setting. This is a multi-attribute problem and it might not affect model too much.

**Comment by abhigoku10** [comment]
does the current code support it ? if not what modifications have to be made
pls share your thoughts

**Comment by FateScript** [comment]
No, you might write such code yourself. Come on, have a try.

**Comment by abhigoku10** [comment]
@FateScript  ya sure will modify it just wanted to knw if its possible with current code , if any references u have pls do share would be helpful

**Comment by mortentang** [comment]
@abhigoku10 Did you manage to implement multi class for same object?

**Comment by abhigoku10** [comment]
@mortentang nope not yet due to other workloads i am not able to spend time on this , do let me knw if you have done it would be good to learn
